service_container/autowiring.rst

```diff
@@ -378,15 +409,22 @@ that alias:
             App\Util\Rot13Transformer: ~
             App\Util\UppercaseTransformer: ~
 
-            # the ``App\Util\Rot13Transformer`` service will be injected when
-            # a ``App\Util\TransformerInterface`` type-hint is detected
+            # the ``App\Util\UppercaseTransformer`` service will be
+            # injected when an ``App\Util\TransformerInterface``
+            # type-hint for a ``$shoutyTransformer`` argument is detected.
```

Comment:
Aren't the comments swapped?

Reply:
Oh shoot, you're right, let me fix this!

Reply:
See https://github.com/symfony/symfony-docs/pull/11389